src/drivers/nrf52/Spi.cpp

```diff
@@ -1,8 +1,8 @@
-#include "drivers/Spi.h"
+#include "Spi.h"
```

Comment:
absolute include paths please
```suggestion
#include "drivers/nrf52/Spi.h"
```